mage/main_test.go

```diff
@@ -31,7 +31,11 @@ func testmain(m *testing.M) int {
 		log.Fatal(err)
 	}
 	if err := os.Mkdir(dir, 0700); err != nil {
-		log.Fatal(err)
+		if os.IsExist(err) {
+			os.RemoveAll(fmt.Sprintf("%s/*", dir))
```

Comment:
RemoveAll removes a directory.  globbing (*) doesn't generally work in go functions anyway.  if you want to just delete everything *inside* the directory, you'll either have to loop through, or better, just delete the whole thing and recreate it.